docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/type/string-tensor-unpack-15.rst

```diff
@@ -0,0 +1,149 @@
+.. {#openvino_docs_ops_type_StringTensorUnpack_15}
+
+StringTensorUnpack
+===================
+
+
+.. meta::
+  :description: Learn about StringTensorUnpack-15 - operation which unpacks a batch of strings into three tensors.
+
+**Versioned name**: *StringTensorUnpack-15*
+
+**Category**: *Type*
+
+**Short description**: *StringTensorUnpack* operation transforms a given batch of strings into three tensors - one containing 
+the concatenated string data, and two other storing begin and end indices of the strings, respectively.
+
+**Detailed description**
+
+Consider an ``input`` string tensor containing values ``["Intel", "OpenVINO"]``.
+
+The operator will transform the tensor into three outputs:
+
+* *begins* = [0, 5]
+    * ``begins[0]`` is equal to 0, because the first string starts at the beginning index.
+    * ``begins[1]`` is equal to 5, because length of the string "Intel" is equal to 5.
+    * ``begins.shape`` is equal to [2], because the ``input`` is a batch of 2 strings.
+
+* *ends* = [5, 13]
+    * ``ends[0]`` is equal to 5, because length of the string "Intel" is equal to 5.
+    * ``ends[1]`` is equal to 13, because length of the string "OpenVINO" is 8, and it needs to be summed up with length of the string "Intel".
+    * ``ends.shape`` is equal to ``[2]``, because the ``input`` is a batch of 2 strings.
+
+* *symbols* = "IntelOpenVINO"
+    * ``symbols`` contains concatenated string data encoded in utf-8 bytes, interpretable using ``begins`` and ``ends``.
+    * ``symbols.shape`` is equal to ``[13]``, because it's the length of concatenated ``input`` strings.
+
+When defining *begins* and *ends*, the notation ``[a, b)`` is used. This means that the range starts with ``a`` and includes all values up to, 
+but not including, ``b``. That is why in the example given the length of "IntelOpenVINO" is 12, but *ends* vector contains 13.
+
+**Inputs**
+
+* **1**: *data*
+
+  * **Description**: A tensor containing a string to be unpacked. **Required.**
+  * **Range of values**: A string tensor.
+  * **Type**: ``string``
+
+**Outputs**
+
+* **1**: *begins*:
+
+  * **Description**: Indices of each string's beginnings.
+  * **Range of values**: ND tensor of non-negative integer numbers.
+  * **Type**: ``int32``
```

Comment:
Same comment about alignment, as for the inputs:

```suggestion
**Outputs**

* **1**: ``begins``  - ND tensor of type ``int32`` and the same shape as the ``data`` input tensor, containing indices (non-negative integer numbers) of each string's beginnings.

```

... for all of the outputs

Reply:
Done.